src/pages/home/sidebar/SidebarScreen.js

```diff
@@ -27,7 +27,7 @@ import {
 } from '../../../components/Icon/Expensicons';
 import Permissions from '../../../libs/Permissions';
 import ONYXKEYS from '../../../ONYXKEYS';
-import {create, isAdminOfFreePolicy} from '../../../libs/actions/Policy';
+import * as Policy from '../../../libs/actions/Policy';
```

Comment:
NAB: Why `*` and not just replace `create` with `createAndNavigate`?

Reply:
More context here: https://github.com/Expensify/App/issues/5984#issuecomment-952302958

Basically we are going to change these all eventually so I got to this one now.